tests/src/Util/HTTPInputDataTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2021, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Test\src\Util;
+
+use Friendica\Test\MockedTest;
+use Friendica\Test\Util\HTTPInputDataDouble;
+use Friendica\Util\HTTPInputData;
+
+/**
+ * Testing HTTPInputData
+ * @see	HTTPInputData
+ */
+class HTTPInputDataTest extends MockedTest
+{
+	/**
+	 * Returns the data stream for the unit test
+	 * Each array element of the first hierarchy represents one test run
+	 * Each array element of the second hierarchy represents the parameters, passed to the test function
+	 * @return array[]
+	 */
+	public function dataStream()
+	{
+		return [
+			'example' => [
```

Comment:
@annando I created a test environment for this class. You now have to replace this array with a valid one. I documented the content of this array as function-documentation.
- Think about each relevant and possible input for this class and create an entry for it
- define the whole input as `input` of each entry
- define the whole output as `expected` of each entry (I thought the whole output in the form of https://github.com/friendica/friendica/pull/10294/files#diff-68f51c6492bf8e899409d8edced368491492558529ab76b92040bd0706defea0R69 would be the most intuitive way, we can adapt it the way you liket it.

Reply:
Where do I store the file with the original content?

Reply:
I created a example file at the right directory and adapted the test case.